### ISSUE TITLE
device_name not working

### ISSUE THREAD

**Issue by thelittlefireman** [custom-components/binary_sensor.hadockermon#5]
**Version of the custom_component**
latest

**Describe the bug**
I try to set the device_name property but all my containers still get hadockermon as prefix name

```
binary_sensor:
  platform: hadockermon
  host: 127.0.0.1
  port: 8126
  device_name: docker
  stats: true
```

**log**

```
Add your logs here.
```

**Comment by ludeeus** [comment]
should be `name:` and not `device_name:` the "docs" are wrong

**Comment by ludeeus** [comment]
Corrected by #6
